### PR TITLE
Fixed cudnn -> nn avg-pooling conversion

### PR DESCRIPTION
**Pull request by lospooky**
cudnn.convert now properly initializes nn.SpatialAveragePooling.count_include_pad

### CODE REVIEW

File: convert.lua [hunk 1]
```diff
@@ -43,7 +43,14 @@ function cudnn.convert(net, dst)
       if v == 'ReLU' then y = dst.ReLU() end -- because parameters
       for k,u in pairs(x) do y[k] = u end
       if src == cudnn and x.clearDesc then x:clearDesc() end
-      if src == cudnn and v == 'SpatialAveragePooling' then y.divide = true end
+      if src == cudnn and v == 'SpatialAveragePooling' then
+        y.divide = true
+        if v.mode == 'CUDNN_POOLING_AVERAGE_COUNT_INCLUDE_PADDING' then
```
Comment (szagoruyko) [inline convert.lua]: can you please do
```lua
y.count_include_pad = v.mode == 'CUDNN_POOLING_AVERAGE_COUNT_INCLUDE_PADDING'
```

Comment (lospooky) [inline convert.lua]: Sure!
Still new to Lua, thanks for the idiomatic tip!

**Comment by szagoruyko** [comment]
conflicts now, can you rebase and squash?

**Comment by lospooky** [comment]
Tried doing it, looks clean from here now.. Let me know!
First time attempting squashing /rebasing stuff...

**Comment by soumith** [comment]
Thanks!

**Pull request merged**